### PR TITLE
testing changes

### PR DESCRIPTION
**Pull request by niveshnagar**
made some random changes

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -95,21 +95,21 @@ const Home: BlitzPage = () => {
                   <div className={styles.code}>
                     <span>1</span>
                     <pre>
-                      <code>blitz generate all project name:string</code>
+                      <code>Green day</code>
```
Comment (varunjram) [inline src/pages/index.tsx:98]: Comment 1

File: src/pages/index.tsx [hunk 2]
```diff
@@ -95,21 +95,21 @@ const Home: BlitzPage = () => {
...
+                      <code>Meow</code>
                     </pre>
                   </div>
 
                   <div className={styles.code}>
                     <span>3</span>
                     <pre>
-                      <code>blitz dev</code>
+                      <code>Pink Floyd</code>
                     </pre>
```
Comment (varunjram) [inline src/pages/index.tsx:113]: Comment 2

**Review by varunjram — COMMENTED** [review]
Changes missing please  change  "Gun n roses" to One Piece

**Comment by varunjram** [comment]
TEST

File: src/pages/index.tsx (hunk 2 above)
Comment (niveshnagar) [reply src/pages/index.tsx:113]: hello

File: src/pages/index.tsx [hunk 3]
```diff
@@ -133,7 +133,7 @@ const Home: BlitzPage = () => {
                   rel="noopener noreferrer"
                   className={styles.card}
                 >
-                  Gun n roses
+                  monkey d luffy
                   <span className={styles.arrowIcon} />
```
Comment (varunjram) [inline src/pages/index.tsx:137]: Done

**Review by varunjram — COMMENTED** [review]
Can be merged

**Pull request merged**